package.json

```diff
@@ -18,34 +18,34 @@
     "test": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^5.1.7"
+    "ember-cli-babel": "^6.6.0"
   },
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.5",
-    "ember-ajax": "^2.4.1",
-    "ember-cli": "~2.11.0",
-    "ember-cli-dependency-checker": "^1.3.0",
-    "ember-cli-eslint": "^3.0.0",
-    "ember-cli-htmlbars": "^1.1.1",
-    "ember-cli-htmlbars-inline-precompile": "^0.3.3",
+    "ember-cli": "~2.16.2",
+    "ember-cli-dependency-checker": "^2.0.0",
+    "ember-cli-eslint": "^4.0.0",
+    "ember-cli-htmlbars": "^2.0.1",
+    "ember-cli-htmlbars-inline-precompile": "^1.0.0",
     "ember-cli-inject-live-reload": "^1.4.1",
     "ember-cli-pretender": "^1.0.1",
-    "ember-cli-qunit": "3.0.1",
-    "ember-cli-test-loader": "^1.1.0",
-    "ember-cli-shims": "^1.0.2",
+    "ember-cli-qunit": "^4.0.0",
+    "ember-cli-shims": "^1.1.0",
     "ember-cli-sri": "^2.1.0",
-    "ember-cli-uglify": "^1.2.0",
-    "ember-data": "2.11.0",
-    "ember-disable-prototype-extensions": "^1.1.0",
-    "ember-export-application-global": "^1.0.5",
+    "ember-cli-test-loader": "^1.1.0",
+    "ember-cli-uglify": "^2.0.0",
+    "ember-data": "2.16.0",
+    "ember-disable-prototype-extensions": "^1.1.2",
+    "ember-export-application-global": "^2.0.0",
     "ember-faker": "^1.1.2",
-    "ember-load-initializers": "^0.6.0",
-    "ember-resolver": "^2.0.3",
-    "ember-source": "~2.11.0",
-    "loader.js": "^4.0.10"
+    "ember-in-viewport": "https://github.com/snewcomer/ember-in-viewport.git#intersection-observer",
```

Comment:
@hhff This is pulling from this [PR](https://github.com/DockYard/ember-in-viewport/pull/113).  This is an exciting change using the IntersectionObserver API (falls back to rAF if not available...then to scroll event handling if rAF is not available)!

Reply:
hahah very impressive, about time something like this existed in the web api!  Nice work with all the OSS Scott!  You're a machine 👍 

conversation over there seems quiet.  

By looking at your PR it seems the concept of a scrolling container that isn't the window isn't supported by ember-in-viewport.  I'd want to be sure we are pulling in an official version of ember-in-viewport rather than a fork.

Also - as an alternative this looks pretty good and supports passing `root` out of the box:
https://github.com/asakusuma/ember-spaniel

Reply:
Yeah probably good to stick with the official version.  Can't really merge this PR w/o a scrolling container.

Yeah I saw this PR so I thought it made sense to integrate (and replace rAF) when it was merged.  But maybe if my PR doesn't get any love, we can run our own implementation with ember-spaniel. 

https://github.com/linkedin/spaniel/pull/63